src/current/cockroachcloud/cluster-management.md

```diff
@@ -68,53 +68,50 @@ To add or remove nodes from your cluster:
 
     The **Edit cluster** page displays.
 
-1. From the **Nodes** dropdown, select the number of nodes you want in each region.
-1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Next: Payment**.
-1. On the **Summary** page, verify your new cluster configuration.
-1. Click **Update**.
+1. On the **Regions** page, select the number of nodes you want in each region from the **Nodes** dropdown.
+1. In the sidebar, verify the hourly estimated cost for the cluster.
+1. Click **Next: Capacity** and then **Next Finalize**.
+1. On the **Finalize** page, click **Update**.
 
-### Increase storage for a cluster
-
-{{site.data.alerts.callout_danger}}
-AWS disks can only be scaled once every six hours.
-{{site.data.alerts.end}}
+### Change compute for a cluster
 
 1. Navigate to the cluster's **Overview** page.
 1. Select **Actions > Edit cluster**.
 
     The **Edit cluster** page displays.
 
-1. Navigate to the **Storage** dropdown in the **Hardware per node** section.
-1. Select the new amount of storage per node.
+1. Navigate to the **Capacity** page.
+1. In the **Compute per node** section, select the new amount of vCPUs per node.
 
-    {{site.data.alerts.callout_danger}}
-    Storage space cannot be removed due to cloud provider limitations.
+    {{site.data.alerts.callout_info}}
+    When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For most production applications, we recommend **at least 4 to 8 vCPUs** per node.
     {{site.data.alerts.end}}
 
-1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Next: Payment**.
-1. On the **Summary** page, verify your new cluster configuration.
-1. Click **Update**.
+1. In the sidebar, verify the hourly estimated cost for the cluster.
+1. Click **Next: Finalize**.
+1. On the **Finalize** page, click **Update**.
 
-### Change compute for a cluster
+### Increase storage for a cluster
+
+{{site.data.alerts.callout_danger}}
+AWS disks can only be scaled once every six hours.
+{{site.data.alerts.end}}
 
 1. Navigate to the cluster's **Overview** page.
 1. Select **Actions > Edit cluster**.
 
     The **Edit cluster** page displays.
 
-1. Navigate to the **Compute** dropdown in the **Hardware per node** section.
-1. Select the new amount of vCPUs per node.
+1. Navigate to the **Capacity** page.
+1. In the **Compute per node** section, select the new amount of storage per node.
```

Comment:
This should say "Storage per node" section

Reply:
Good catch